doc/monitoring/getting_started.rst

```diff
@@ -1,68 +1,140 @@
 
-    httpd.new('0.0.0.0', 8088)
-        :route({path = '/metrics'}, function(...)
-            return http_handler(...)
-    end)
-        :start()
+#.  Start the HTTP server and expose metrics:
 
+    ..  code-block:: lua
+
+        local http_server = require('http.server')
+        local server = http_server.new('0.0.0.0', 8081)
+        server:route({path = '/metrics'}, http_metrics_handler)
+        server:start()
+
+The metric values are now available via the ``http://localhost:8081/metrics`` URL:
+
+..  code-block:: json
+
+    [
+      {
+        "label_pairs": {
+          "alias": "my-instance"
+        },
+        "timestamp": 1679663602823779,
+        "metric_name": "tnt_vinyl_disk_index_size",
+        "value": 0
+      },
+      {
+        "label_pairs": {
+          "alias": "my-instance"
+        },
+        "timestamp": 1679663602823779,
+        "metric_name": "tnt_info_memory_data",
+        "value": 39272
+      },
+      {
+        "label_pairs": {
+          "alias": "my-instance"
+        },
+        "timestamp": 1679663602823779,
+        "metric_name": "tnt_election_vote",
+        "value": 0
+      }
+    ]
+
+The data can be visualized in
+`Grafana dashboard <https://www.tarantool.io/en/doc/latest/book/monitoring/grafana_dashboard/#monitoring-grafana-dashboard-page>`__.
+
+The full source example is listed below:
+
+..  code-block:: lua
+
+    -- Import modules
+    local metrics = require('metrics')
+    local http_server = require('http.server')
+    local json_exporter = require('metrics.plugins.json')
+
+    -- Define helper functions
+    local function http_metrics_handler(request)
+        return request:render({ text = json_exporter.export() })
+    end
+
+    -- Start the database
     box.cfg{
-        listen = 3302
+        listen = 3301,
     }
 
-Now you can use the HTTP API endpoint ``/metrics`` to collect your metrics
-in the Prometheus format. To learn how to obtain custom metrics, check the
-:ref:`API reference <metrics-api_reference>`.
+    -- Configure the metrics module
+    metrics.cfg(labels = {alias = 'my-tnt-app'})
+
+    -- Run the web server
+    local server = http_server.new('0.0.0.0', 8081)
+    server:route({path = '/metrics'}, http_metrics_handler)
+    server:start()
 
 ..  _monitoring-getting_started-http_metrics:
 
-Collect HTTP metrics
---------------------
+Collecting HTTP metrics
+-----------------------
 
-To enable the collection of HTTP metrics, you need to create a collector first.
+To enable the collection of HTTP metrics, wrap a handler with a ``metrics.http_middleware.v1`` function:
 
 ..  code-block:: lua
 
+    local metrics = require('metrics')
     local httpd = require('http.server').new(ip, port)
 
     -- Create a summary collector for latency
-    local collector = metrics.http_middleware.build_default_collector('summary')
+    local default_collector = metrics.http_middleware.build_default_collector('summary')
```

Comment:
You can use single `configure_default_collector` instead of `build`+`set`

Reply:
Fixed